atom/common/api/atom_api_native_image.cc

```diff
@@ -22,6 +22,10 @@
 #include "ui/gfx/geometry/size.h"
 #include "ui/gfx/image/image_skia.h"
 #include "ui/gfx/image/image_util.h"
+#if defined(OS_WIN)
```

Comment:
Add a blank line before this statement. The platform-related headers should be guarded as an independent part.